tests/sentry/backup/test_imports.py

```diff
@@ -81,6 +85,14 @@ def export_to_tmp_file_and_clear_database(self, tmp_dir) -> Path:
         clear_database()
         return tmp_path
 
+    def import_empty_backup_file(self, import_fn):
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            tmp_empty_file_path = tmp_dir + "empty_backup.json"
+        with open(tmp_empty_file_path, "w") as tmp_file:
+            json.dump([], tmp_file)
+        with open(tmp_empty_file_path, "rb") as empty_backup_json:
+            import_fn(empty_backup_json, printer=NOOP_PRINTER)
```

Comment:
I think these lines should be indented so that they remain in the context of `tmp_dir`, otherwise it is possible for the new directory to be deleted before this writes/reads can occur.

Reply:
good ctatch